### ISSUE TITLE
Problems with filter property in scss.

### ISSUE THREAD

**Issue by GoliafRS** [brackets-beautify/brackets-beautify#214]
Before Beautify:
`filter: progid:DXImageTransform.Microsoft.gradient(***);`

After Beautify:
`filter: progid: DXImageTransform.Microsoft.gradient(***);`

Space after progid causes an error at compile scss to css.

**Comment by bitwiseman** [comment]
This is fixed in js-beautify 1.6.4.

**Comment by Hirse** [comment]
@GoliafRS Please try [v2.3.0](https://github.com/brackets-beautify/brackets-beautify/releases/tag/v2.3.0) which includes an updated js-beautify library.

**Comment by archi-tec** [comment]
Hello,
I updated my beautify to 2.3.0, but the problem is still here.

**Comment by Hirse** [comment]
@archi-tec Please provide an example for your code and actual and expected output.

**Comment by archi-tec** [comment]
@Hirse here is it:

Before
`filter: progid:DXImageTransform.Microsoft.Blur(PixelRadius='3');`

After
`filter: progid: DXImageTransform.Microsoft.Blur(PixelRadius='3');`

**Comment by GoliafRS** [comment]
I confirm that the problem is still there.

Before
`filter: progid:DXImageTransform.Microsoft.gradient(startColorstr='#000000', endColorstr='#000000', GradientType=1);`

After
`filter: progid: DXImageTransform.Microsoft.gradient(startColorstr='#000000', endColorstr='#000000', GradientType=1);`

**Comment by bitwiseman** [comment]
@GoliafRS @archi-tec 
Could you show a but more of the code around these lines? 
Context matters.

**Comment by archi-tec** [comment]
@bitwiseman  Here is below. One of these rules is also inside of media query @media(min-width:768px) { ... }, but two others - not.

This is css file, however in scss there is the same issue.
Let me know if you need more.

```
.miami.about-pf16 .blurred-background-container img {
    -webkit-filter: blur(4px);
    -moz-filter: blur(4px);
    -ms-filter: blur(4px);
    -o-filter: blur(4px);
    /* FF doesn't support blur filter, but SVG */
    filter: url("data:image/svg+xml;utf8,<svg height='0' xmlns='http://www.w3.org/2000/svg'><filter id='svgBlur' x='-4%' y='-4%' width='110%' height='110%'><feGaussianBlur in='SourceGraphic' stdDeviation='4'/></filter></svg>#svgBlur");
    filter: progid:DXImageTransform.Microsoft.Blur(PixelRadius='4');
    filter: blur(4px);
}
```

**Comment by bitwiseman** [comment]
And the above turns into this? 

```
.miami.about-pf16 .blurred-background-container img {
    -webkit-filter: blur(4px);
    -moz-filter: blur(4px);
    -ms-filter: blur(4px);
    -o-filter: blur(4px);
    /* FF doesn't support blur filter, but SVG */
    filter: url("data:image/svg+xml;utf8,<svg height='0' xmlns='http://www.w3.org/2000/svg'><filter id='svgBlur' x='-4%' y='-4%' width='110%' height='110%'><feGaussianBlur in='SourceGraphic' stdDeviation='4'/></filter></svg>#svgBlur");
    filter: progid: DXImageTransform.Microsoft.Blur(PixelRadius='4');
    filter: blur(4px);
}
```

**Comment by archi-tec** [comment]
yes

```
.miami.about-pf16 .blurred-background-container img {
    -webkit-filter: blur(4px);
    -moz-filter: blur(4px);
    -ms-filter: blur(4px);
    -o-filter: blur(4px);
    /* FF doesn't support blur filter, but SVG */
    filter: url("data:image/svg+xml;utf8,<svg height='0' xmlns='http://www.w3.org/2000/svg'><filter id='svgBlur' x='-4%' y='-4%' width='110%' height='110%'><feGaussianBlur in='SourceGraphic' stdDeviation='4'/></filter></svg>#svgBlur");
    filter: progid: DXImageTransform.Microsoft.Blur(PixelRadius='4');
    filter: blur(4px);
}
```

**Comment by bitwiseman** [comment]
@archi-tec 
Excellent.  I can reproduces this.  And I found the bug filed for it (that has not been fixed). 
https://github.com/beautify-web/js-beautify/issues/755

**Comment by Hirse** [comment]
As the js-beautify issue is closed for js-beautify 1.6.5 I am closing this with version 2.4.0.
